### DISCUSSION TITLE
Required crnn_vgg16_bn onnx files

### DISCUSSION THREAD

**Discussion by Shiva-Ai4m** [mindee/doctr#1245] [General]
I require crnn_vg166_bn.onnx model for conversion.I need tensorflow model to be connverted into onnx model .   I have been trying for 2 days but i am unsucessful in changing it. please can anyone provide me the onnx model. @felixT2K

**Comment by felixT2K** [comment]
Take a look at: https://github.com/mindee/doctr/blob/e04e183be04abf56217a50797a0d3e951dddb6dc/tests/tensorflow/test_models_recognition_tf.py#L172

But keep in mind the model is exported up to the logits so postprocessing is on your side (for the moment - a `onnx_predictor` is o track`

**Comment by felixT2K** [comment]
```
input_shape = (32, 128, 3)
model = recognition.crnn_vgg16_bn(pretrained=True, exportable=True, input_shape=input_shape)
dummy_input = [tf.TensorSpec([None, *input_shape], tf.float32, name="input")]
model_path, output = export_model_to_onnx(model, model_name="crnn_vgg16_bn",  dummy_input=dummy_input)
```
